DebugLog.h

```diff
@@ -21,6 +21,12 @@ using DebugLogLevel = arx::debug::LogLevel;
 using DebugLogBase = arx::debug::LogBase;
 using DebugLogPrecision = arx::debug::LogPrecision;
 
+// Defualt log level to INFO
+#if !defined(DEBUGLOG_DEFAULT_LOG_LEVEL_ERROR) && !defined(DEBUGLOG_DEFAULT_LOG_LEVEL_WARN) && !defined(DEBUGLOG_DEFAULT_LOG_LEVEL_INFO) && !defined(DEBUGLOG_DEFAULT_LOG_LEVEL_DEBUG) && !defined(DEBUGLOG_DEFAULT_LOG_LEVEL_TRACE)
+  #warning "Defaulting log level to: DEBUGLOG_DEFAULT_LOG_LEVEL_INFO"
+  #define DEBUGLOG_DEFAULT_LOG_LEVEL_INFO
+#endif
+
```

Comment:
We have `LOG_SET_LEVEL(level)` now, so this is not necessary for this pr (this p-r is for `LOG_PREAMBLE`, I think)